src/interactive-window/commands/commandRegistry.ts

```diff
@@ -479,6 +482,15 @@ export class CommandRegistry implements IDisposable {
         }
     }
 
+    private async createNewNotebook(): Promise<void> {
+        this.appShell
+            .showInformationMessage(
+                'This command has been deprecated and will eventually be removed, please use "Create: New Jupyter Notebook" instead'
+            )
+            .then(noop, noop);
+        await this.nativeNotebookCreator.createNewNotebook();
```

Comment:
Can't we just call the VS Code command here directly `commands.executeCommand(<new VS Code Command Id>)`
this way we do'nt need nativeNotebookCreator

Reply:
I thought based on the standup discussion today, we'd display the message and then run the new VS Code command to create the jupyter notebook

Reply:
yeah that's a good idea, then I can drop the rest of that stuff again